Gopkg.toml

```diff
@@ -90,11 +90,6 @@
   name = "github.com/tendermint/btcd"
   revision = "e5840949ff4fff0c56f9b6a541e22b63581ea9df"
 
-# Haven't made a release since 2016.
```

Comment:
Ideally the Gopkg.toml lists explicit versions of all dependencies.

Nice to see that this project cut new releases recently. Instead of removing this entry outright, can we move it to the section above (for repos that have releases) and pin it to the latest release version? Looks like that would be v0.9.1

Reply:
> The issue here is we pinned to an outdated version of a dependencies dependency

Oh? `github.com/prometheus/client_golang` is a direct dependency of Tendermint code.

Reply:
oh, I didn't know that dep would let you use transitive dependencies correctly! I wonder if theres a way to just block that directly.